static/src/inline-svgs/icon/mail.svg

```diff
@@ -1 +1 @@
-<svg width="24" height="24" viewBox="0 0 24 24"><path d="M17.5 10.3l-.3-.3-5 2.5L7 10l-.2.3L12 15l5.5-4.7zm1-4.3H5.4L4 8.3V17l1.4 1h13.3l1.3-1V8.2L18.6 6zm-.4 10H6V8h12v8z"/></svg>
+<svg width="14" height="14" viewBox="0 0 14 14"><path d="M-276.2 395.7l-.3-.3-4.4 2.2-4.5-2.2-.2.3 4.5 4.1 4.9-4.1zm.9-3.8h-11.5l-1.2 2v7.6l1.2.9h11.6l1.1-.9v-7.7l-1.2-1.9zm-.4 8.8h-10.6v-7h10.5l.1 7z"/></svg>
```

Comment:
Yep, in cropping these I discovered the mail icon was misaligned and replaced it with the correct one (Sorry should have mentioned). Here they are overlayed to illustrate the inconsistency: 

![screen shot 2015-10-08 at 15 20 03](https://cloud.githubusercontent.com/assets/14570016/10369053/22da69a6-6dd0-11e5-9986-10bebd4bab43.png)